playwright/tests/create-process.test.ts

```diff
@@ -38,23 +37,27 @@ test.describe('Create Process', () => {
   });
 
   test('Add business process and execute it', async () => {
+    await processEditor.hasNoStatusMessage();
     await explorer.addProcess(projectName, processName, 'Business Process');
-    await explorer.hasNode(`${processName}.p.json`);
     await explorer.hasStatusMessage('Finished: Deploy Ivy Projects');
+    await explorer.hasNode(`${processName}.p.json`);
     const start = processEditor.locatorForElementType('g.start\\:requestStart');
     const end = processEditor.locatorForElementType('g.end\\:taskEnd');
     await processEditor.startProcessAndAssertExecuted(start, end);
   });
 
   test('Assert that process gets redeployed after editing', async () => {
+    await processEditor.hasNoStatusMessage();
     await explorer.addProcess(projectName, processName, 'Business Process');
-    await explorer.hasNoStatusMessage();
+    await explorer.hasStatusMessage('Finished: Deploy Ivy Projects');
```

Comment:
maybe this status message check should be part of the addProcess method?

Reply:
or at least the explorer should have a method that do this stuff, instead of duplicate the message here x times and if we change it in the future we need to change it everywhere.